core/src/util/builders.cpp

```diff
@@ -288,103 +288,123 @@ bool isOutsideTile(const glm::vec3& _a, const glm::vec3& _b) {
     return false;
 }
 
-void buildPolyLineSegment(const Line& _line, PolyLineBuilder& _ctx) {
+glm::vec2 processPoint(const Point& coordCurr, const Point& coordNext, const glm::vec2& normPrev,
```

Comment:
It's not at all clear what this function does or what it returns. Naming is hard, so keeping things inline where they are used can be a good strategy to minimize the amount of naming that one has to do :)